### PR TITLE
feat(arukereso): Add GDS entry for arukereso

### PR DESCRIPTION
**Pull request by olof-funnel**

### CODE REVIEW

**Comment by olof-funnel** [comment]
Feel free to squash if merging

File: sources/top_cashback.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "TOP_CASHBACK",
```
[the remark this answers is not in the document]
Comment (olof-funnel) [reply sources/top_cashback.json]: They are called `TopCashback` so no space but seems like two words, but will go with no underscore then 👍

File: sources/u_gift_ideas.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "U_GIFT_IDEAS",
```
[the remark this answers is not in the document]
Comment (olof-funnel) [reply sources/u_gift_ideas.json]: 👍

**Comment by norric-funnel** [comment]
Upstream PR in https://github.com/googledatastudio/ds-data-registry/pull/116

**Pull request closed without merging**